.github/workflows/Check-References.ps1

```diff
@@ -0,0 +1,44 @@
+﻿. $PSScriptRoot"\Get-Package-Version.ps1"
+
+function Check-References([string] $RootDirectory, [string] $PackageName, [string[]] $VersionMustContain, [string[]] $VersionMustNotContain) {
+    # Find all of the packages.config files
+    $packageFiles = @(Get-ChildItem -Path $RootDirectory -Filter packages.config -Recurse | Select-Object -ExpandProperty "FullName")
+    
+	# The value we will return to indicate whether the references as valid or not
```

Comment:
Misaligned

Reply:
Bit strange that gh doesn't update the source file in this view after a push (just says 'outdated'), but it's fixed.